tests/test_resources/test_modules/test_functions/test_aws_lambda/test_lambdas_with_fn.py

```diff
@@ -1,10 +1,56 @@
+import os
+from pathlib import Path
+
+
 def summer(a, b):
     return a + b
 
 
+def multiplier(a: int, b: int):
+    return a * b
+
+
 def test_create_func():
     import runhouse as rh
 
     aws_func = rh.aws_lambda_function(fn=summer, name="summer_lambdas").save()
     res = aws_func(1, 3)
     assert int(res) == 4
+    try:
+        os.remove(
+            str(Path(aws_func.handler_path).parent / f"rh_handler_{aws_func.name}.py")
+        )
+    except FileNotFoundError:
+        assert True is True
+    try:
+        os.remove(
+            str(Path(aws_func.handler_path).parent / f"{aws_func.name}_code_files.zip")
+        )
+    except FileNotFoundError:
+        assert True is True
```

Comment:
if we expect an exception to be raised, would be better to do something like: 
```
    with pytest.raises(Exception): 
        rh.aws_lambda_function()
```

Reply:
Done